specification/desktopvirtualization/resource-manager/Microsoft.DesktopVirtualization/preview/2021-01-14-preview/desktopvirtualization.json

```diff
@@ -3870,6 +3870,58 @@
         "updateErrorMessage": {
           "type": "string",
           "description": "The error message."
+        },
+        "sessionHostHealthCheckResults": {
+          "type": "array",
+          "readOnly": true,
+          "description": "List of SessionHostHealthCheckReports",
+          "items": {
+            "$ref": "#/definitions/SessionHostHealthCheckReport"
+          }
+        }
+      }
+    },
+    "SessionHostHealthCheckReport": {
+      "type": "object",
+      "description": "The report for session host information.",
+      "properties": {
+        "healthCheckName": {
+          "type": "string",
+          "description": "Represents the name of the health check operation performed.",
+          "readOnly": true
+        },
+        "healthCheckResult": {
+          "type": "string",
+          "description": "Represents the Health state of the health check we performed.",
+          "readOnly": true
+        },
+        "additionalFailureDetails": {
+          "$ref": "#/definitions/SessionHostHealthCheckFailureDetails",
+          "description": "Additional detailed information on the failure.",
+          "readOnly": true
+        }
+      }
+    },
+    "SessionHostHealthCheckFailureDetails": {
+      "type": "object",
+      "description": "Contains details on the failure.",
+      "properties": {
+        "message": {
+          "type": "string",
+          "description": "Failure message: hints on what is wrong and how to recover.",
+          "readOnly": true
+        },
+        "errorCode": {
+          "type": "integer",
+          "format": "int32",
+          "description": "Error code corresponding for the failure.",
+          "readOnly": true
+        },
+        "lastHealthCheckInUTC": {
```

Comment:
Please use property name 'lastHealthCheckDateTime'. Refer naming guidelines here: https://github.com/microsoft/api-guidelines/blob/vNext/Guidelines.md#176-date-and-time-properties

Reply:
ok

Reply:
I still see the same property name (lastHealthCheckInUTC and not the recommended one).